iOS_Project/Module/Home/View/HomeViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  HomeViewController.swift
+//  iOS_Project
+//
+//  Created by Asalah Sayed on 25/05/2023.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController , UITableViewDelegate,UITableViewDataSource, UICollectionViewDelegate,UICollectionViewDataSource{
+    
+    @IBOutlet weak var categoryCollection: UICollectionView!
+    @IBOutlet weak var catergoryMeals: UITableView!
+    let caterNames = ["Popular","Breakfast","Lunch","Dinner","Dessert"]
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "Food Recipies"
+        
+
+        
+    }
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        3
+        
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "FavouriteTableViewCell", for: indexPath) as! FavouriteTableViewCell
```

Comment:
please do not use force Unwrap here use OPTIONAL

Reply:
resolved